ReText/window.py

```diff
@@ -217,6 +217,8 @@ def __init__(self, parent=None):
 			lambda: self.searchBar.setVisible(False),
 			shct=QKeySequence.Cancel)
 		self.actionCloseSearch.setPriority(QAction.LowPriority)
+		self.actionInsertImage = self.act(self.tr('Image'), 'image',
```

Comment:
This will be easy to confuse with the existing “Paste image” option, which allows pasting from clipboard.

Maybe rename to something like “Image by file path”?

Reply:
Such as to insert image will go to Edit menu, I will put as "Insert images by file path"

Reply:
OK. Maybe in singular? “Insert *image* by file path”.

Reply:
Sure. But now, I'm using the `getOpenFileNames` method to allows get more than one image. Still use *image* in singular?

Reply:
In this diff, you are using `getOpenFileName`, not `getOpenFileNames`…

Reply:
For now, I'm using `getOpenFileNames`